### PR TITLE
London9 Lovelace - Oleksii Chepurnyi - JScriptCore 2 Coursework - Week 4

### PR DESCRIPTION
**Pull request by OleksChep**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Oleksii Chepurnyi
- Your City:  London
- Your Slack Name: Oleks

# Homework Details

- Module: JavaScript Core 2 Coursework
- Week: 4

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: 1-mandatory/1-alarmclock/alarmclock.js [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
+  }:${seconds < 10 ? "0" : ""}${seconds}`;
+
+  intervalId = setInterval(() => {
+    if (remainingTime > 0) {
+      remainingTime--;
+      let minutes = Math.floor(remainingTime / 60);
+      let seconds = remainingTime % 60;
+      document.getElementById("timeRemaining").innerText = `Time Remaining: ${
+        minutes < 10 ? "0" : ""
+      } : ${seconds < 10 ? "0" : ""}${seconds}`;
```
Comment (LorenaCapraru) [inline 1-mandatory/1-alarmclock/alarmclock.js:21]: Good way of representing the time remaining. You just forgot the minutes

`Time Remaining: ${
        minutes < 10 ? "0" : ""
      } ${minutes}: ${seconds < 10 ? "0" : ""}${seconds}`;

**Comment by LorenaCapraru** [comment]
Very good your approach for Alarm Clock.  Very easy to follow and descriptive. Good job, Olek!

File: 1-mandatory/1-alarmclock/alarmclock.js (hunk 1 above)
Comment (OleksChep) [reply 1-mandatory/1-alarmclock/alarmclock.js:21]: Ok,  thank you, Lorena, I'll fix it now.

**Comment by OleksChep** [comment]
Lorena, thanks for your review, you have a lot to learn!

**Pull request closed without merging**